analyzer/library/pip/pip.go

```diff
@@ -0,0 +1,44 @@
+package pip
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/aquasecurity/fanal/analyzer"
+	"github.com/aquasecurity/fanal/analyzer/library"
+	"github.com/aquasecurity/fanal/types"
+	"github.com/aquasecurity/fanal/utils"
+	"github.com/aquasecurity/go-dep-parser/pkg/pip"
+	"golang.org/x/xerrors"
+)
+
+func init() {
+	analyzer.RegisterAnalyzer(&pipLibraryAnalyzer{})
+}
+
+const version = 1
+
+var requiredFiles = []string{"requirements.txt"}
```

Comment:
We don't have a plan to add more files.

```suggestion
var requiredFile = "requirements.txt"
```

Reply:
fixed